0_hello_world/main.go

```diff
@@ -0,0 +1,13 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, "Hallo du schöne Welt!")
```

Comment:
So crazy, it might just work. The danger is other packages reading and storing os.Stdout on startup. The definitive approach is to build and run with stdin/stdout pipes.

Reply:
@anzboi, @anzdaddy, @camh-anz  - I read that stackoverflow too, but it's kinda not [recommended practice](https://stackoverflow.com/a/10474375) @anzdaddy, I'm not really sure what you mean by the "definitive approach" - I'd love to know though.

Also, please remember this meant to be `go-training` - I don't want to overwhelm people with pipes etc.
All I want is 100% coverage. In the real world we wouldn't work with stdout in this way.

I do agree that `out` as a private global var is not ideal, how about I add a comment as a compromise? 
Or do you have any other good ideas on how to achieve 100% overage?